docker-compose-internal.yaml

```diff
@@ -24,6 +24,9 @@ services:
     build:
       context: .
       dockerfile: node/cmd/Dockerfile
+      platforms:
+        - "linux/amd64"
+        - "linux/arm64"
```

Comment:
Should we add arm images to the other components too?

Reply:
Dropping this from this change. Building with docker compose is kludgy and we might be better off using buildx directly. There is also a problem with gitversion that needs more debugging.